app/models/person.rb

```diff
@@ -6,8 +6,10 @@ class Person < ApplicationRecord
   has_many :badge_assignments
   has_many :badges, through: :badge_assignments
   has_many :tokens, as: :tokenable
+  has_many :account_relationships, as: :holder
+  has_many :accounts, through: :account_relationships
   before_save :check_display_name
-  accepts_nested_attributes_for :user, reject_if: proc { |attribute| attribute['password'].empty? }
```

Comment:
This is completely unrelated to the accounts work, but you can't update a user without setting their password with this in place.

Reply:
I think this was to solve a issue that kept occurring when I was implementing #129. I'll look into it later and try and find another patch.